packages/dashboard/src/customization/widgets/lineScatterChart/component.tsx

```diff
@@ -25,6 +25,8 @@ import NoChartData from '../components/no-chart-data';
 import { default as lineSvgDark } from './line-dark.svg';
 import { IoTSiteWiseDataStreamQuery } from '~/types';
 import { assetModelQueryToSiteWiseAssetQuery } from '../utils/assetModelQueryToAssetQuery';
+// eslint-disable-next-line no-restricted-imports
+import { useVisualizedDataStreams } from '@iot-app-kit/react-components/src/components/chart/hooks/useVisualizedDataStreams';
```

Comment:
why the eslint exceptions ? you should not be importing from src, if you need this exposed, updated the react-components public exports file

Reply:
Resolved